src/main/java/com/bookstore/service/BookServiceImpl.java

```diff
@@ -0,0 +1,41 @@
+package com.bookstore.service;
+
+import com.bookstore.dto.BookDto;
+import com.bookstore.dto.CreateBookRequestDto;
+import com.bookstore.exception.EntityNotFoundException;
+import com.bookstore.mapper.BookMapper;
+import com.bookstore.entity.Book;
+import lombok.RequiredArgsConstructor;
+import org.springframework.stereotype.Service;
+import com.bookstore.repository.BookRepository;
+
```

Comment:
I think checkstyle doesn't work, check it please and remove line

Reply:
I check with the command mvn checkstyle:check 
maybe it's not the right version of the plugin?